app/routes/orders/view.js

```diff
@@ -13,15 +13,18 @@ export default class ViewRoute extends Route {
   }
 
   async model(params) {
+
     const order = await this.store.findRecord('order', params.order_id, {
       include : 'attendees,tickets,event',
       reload  : true
     });
     const eventDetails = await order.query('event', { include: 'tax' });
+
     return {
       order,
-      event : eventDetails,
-      form  : await eventDetails.query('customForms', {
+      event      : eventDetails,
+      taxDetails : await eventDetails.get('tax', { cache: true, public: true }),
+      form       : await eventDetails.query('customForms', {
```

Comment:
Remove await and use hash function to wrap the return object

Reply:
yes sir

Reply:
if i will use hash , then i should remove every await statements

Reply:
inside the return object, yes